src/driver/include/driver/DefaultDriver.hpp

```diff
@@ -47,18 +63,8 @@ class DefaultDriver {
         std::string metricsOutputFileName;
         std::string mongoUri;
         std::string description;
-        bool isHelp = false;
-        bool shouldListActors = false;
-        bool isDryRun = false;
-    };
-
-
-    enum class OutcomeCode {
-        kSuccess = 0,
-        kStandardException = 1,
-        kBoostException = 2,
-        kInternalException = 3,
-        kUnknownException = 10,
+        bool showHelp = false;
+        DefaultDriver::RunMode runMode = RunMode::kNormal;
```

Comment:
sorry to keep iterating but does `showHelp` also want to be a run-mode?

Reply:
Yeah that makes sense. I didn't know you could do `git help` or `brew help`